client/gutenberg/extensions/contact-info/style.scss

```diff
@@ -0,0 +1,3 @@
+.wp-block-jetpack-contact-info {
+	margin-bottom: 1.75em;
```

Comment:
Can we change this to `1.5em` so we have some consistency with our other blocks.

Reply:
Done :)